filter_processed.py

```
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import csv
+import sys
+
+def filter_unprocessed_candidates(input_csv, output_csv, processed_file):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)